src/index.ts

```diff
@@ -126,6 +130,12 @@ class Monika extends Command {
       default: false,
     }),
 
+    prometheus: flags.integer({
```

Comment:
If the user does not specify the port number, at what port did prometheus start?

Reply:
If the user doesn't specify the port, Monika won't start. The user has to explicitly set the port. Do you have any suggestions?

Reply:
I think it's better that way.